pkg/controller/user/user_controller.go

```diff
@@ -17,7 +16,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller_user")
+var (
+	log = logf.Log.WithName("controller_user")
+	// A set of pre configured groups used to exclude a user from rhmi specific groups
+	exclusionGroups = []string{
+		"layered-cs-sre-admins",
+		"osd-sre-admins",
+	}
```

Comment:
Looks to be the `exclusionGroups` in common in `user_controller ` and `rhsso` reconciler, what do you think on moving this to `userHelper.go`?